src/test/java/com/puppycrawl/tools/checkstyle/checks/ArrayTypeStyleCheckTest.java

```diff
@@ -68,15 +68,15 @@ public void testJavaStyleOff()
             createModuleConfig(ArrayTypeStyleCheck.class);
         checkConfig.addAttribute("javaStyle", "false");
         final String[] expected = {
-            "13:16: " + getCheckMessage(MSG_KEY),
-            "17:39: " + getCheckMessage(MSG_KEY),
-            "23:18: " + getCheckMessage(MSG_KEY),
-            "31:20: " + getCheckMessage(MSG_KEY),
-            "45:33: " + getCheckMessage(MSG_KEY),
-            "50:36: " + getCheckMessage(MSG_KEY),
-            "56:29: " + getCheckMessage(MSG_KEY),
+            "12:16: " + getCheckMessage(MSG_KEY),
```

Comment:
input has 4 `// violation` comments but 7 violations are expected, please check and fix

Reply:
@timurt the first 4 lines are `// violation`, but the last 3 lines are `// ok`

Please check again. I think the last 3 lines are actually in C style.